File: src/main/java/no/nav/syfo/domain/Arbeidsoppgave.java
package no.nav.syfo.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class Arbeidsoppgave {
    public Long id;
    public long oppfoelgingsdialogId;
    public String navn;

    public boolean erVurdertAvSykmeldt;
    public Gjennomfoering gjennomfoering = new Gjennomfoering();
    public String sistEndretAvAktoerId;
    public LocalDateTime sistEndretDato;
    public String opprettetAvAktoerId;
    public LocalDateTime opprettetDato;

}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfolgingsplan/Arbeidsoppgave.kt
package no.nav.syfo.api.v3.domain.oppfolgingsplan

import java.time.LocalDateTime

data class Arbeidsoppgave (
    val arbeidsoppgaveId: Long,
    val arbeidsoppgavenavn: String,
    val erVurdertAvSykmeldt : Boolean,
    val gjennomfoering: Gjennomfoering?,
    val opprettetDato: LocalDateTime,
    val sistEndretDato: LocalDateTime,
    val sistEndretAv: Person,
    val opprettetAv: Person,
)
